src/utils/components/CoordinateCopySlot.vue

```diff
@@ -23,15 +24,21 @@ const props = defineProps({
         type: Number,
         default: 1000,
     },
+    coordinateFormat: {
```

Comment:
Here either use the the format object as input or the id. I would use the object, which means you should have

```
 coordinateFormat: {
        type: CoordinateFormat,
        default: null,
    },
```